packages/storybook/src/migrations/update-10-3-0/update-10-3-0.ts

```diff
@@ -53,11 +54,12 @@ function updateLintTarget(
   const architect = options.projectConfig.architect;
 
   const paths = {
-    tsConfig: path.join(options.projectConfig.root, 'tsconfig.json'),
-    tsConfigLib: path.join(options.projectConfig.root, 'tsconfig.lib.json'),
-    tsConfigStorybook: path.join(
-      options.projectConfig.root,
-      '.storybook/tsconfig.json'
+    tsConfig: normalize(path.join(options.projectConfig.root, 'tsconfig.json')),
```

Comment:
This is required because the project root has `\\` in its path?

Reply:
no, there was an issue that Rares mentioned related to these paths on Windows. I'm working on a fix just now

Reply:
but to be precise, that specific line wouldn't probably need the normalize, I'm gonna remove it there